onnx/optimizer/optimize.h

```diff
@@ -31,6 +31,8 @@ struct Optimizer final {
     _registerOptimizer<SplitPredict>();
   }
 
+  virtual ~Optimizer() = default;
```

Comment:
curious, why there is a dup above
```
_registerOptimizer<FuseConsecutiveTransposes>();
_registerOptimizer<FuseConsecutiveTransposes>();
```
any specific reason?

Reply:
nope. Thanks for catching.